lesson_02/job1/tests/test_storage.py

```diff
@@ -0,0 +1,65 @@
+"""
+Tests api.py
+# TODO: write tests
+"""
+import json
+import os.path
+from unittest import TestCase, mock
+from lesson_02.job1.tests.common import remove_dir
+from lesson_02.job1.storage import save_to_disk
+
+class SaveToDiskTestCase(TestCase):
+    temp_dir = '/tmp/de07/tests/raw/sales'
+    test_data = [{
+                "key1": "value1",
+                "key2": "value2",
+                "key3": "value3"
+            },
+            {
+                "key4": "value4",
+                "key5": "value5",
+                "key6": "value6"
+            }]
+    """
+    Test storage.save_to_disk function.
+    """
+    def tearDown(self) -> None:
+        """
+        Here we just delete all the temporary paths etc.
+        """
+        remove_dir(self.temp_dir)
+
+    def read(self, path):
+        data = None
+        with open(path, 'rt+') as f:
+            data = json.load(f)
+        return data
+
+    def testWriteCorrectData(self):
+        #verify writing to the path
+        path = os.path.join(self.temp_dir, 'test.json')
+        save_to_disk(self.test_data, path)
+        readData = self.read(path)
+        self.assertIsNotNone(readData)
```

Comment:
use blank lines between different code sections. 
For instance, I would use a blank line between the preparation section and the assert section.

Code should be like a poem, without blank lines it looks like a mess